assets/js/components/OutfrontTakeoverTool/AlertDashboard/PastAlertsList.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable react/prop-types */
```

Comment:
`:oof:` but understandable for now...

Reply:
Another solution is to pull in the `prop-types` library so that we can validate the props for these class components. I'm happy to go that direction if it is preferred.

Reply:
I think they definitely are being type-checked as we have the appropriate `extends React.Component<...>`, but... maybe the linter doesn't understand that? I'm looking through all the [examples](https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/prop-types.md) and the only ones where they use class components are the non-TypeScript ones. Weird gap in the functionality, if true.